admin/google_search_console/class-gsc-issues.php

```diff
@@ -99,9 +99,12 @@ private function save_fetched_issues( array $fetched_issues ) {
 	 * @param stdClass $issue        Issue object to check against the list.
 	 */
 	private function issue_compare( &$crawl_issues, $issue ) {
-		$issue->pageUrl = WPSEO_Utils::format_url( (string) $issue->pageUrl );
+		// Ignore coding standards for object properties.
+		// @codingStandardsIgnoreStart
+		$page_url = $issue->pageUrl = WPSEO_Utils::format_url( (string) $issue->pageUrl );
+		// @codingStandardsIgnoreEnd
```

Comment:
Those properties should pass as long as they are on the whitelist as added in https://github.com/Yoast/wordpress-seo/pull/6875/commits/14f775fc7a7342dbf14e42a93870bf457b679d36